app/main.py

```diff
@@ -2,0 +1,15 @@
+
+def copy_file(command: str) -> None:
+    command_parsed = command.split(" ")
+    file_in_name = command_parsed[1]
+    file_out_name = command_parsed[2]
+
+    if command_parsed[0] == "cp":
```

Comment:
This line checks if the first element of the command is 'cp'. However, this check is not necessary because the function is specifically designed to handle 'cp' commands. Removing this check would make the code cleaner.